### PR TITLE
ECCI-464/change-position-of-newsroom-page-components

### PR DESCRIPTION
**Pull request by kurtisRogers-Nomensa**
### What did I _**try**_ to do?
- Managed to move the newsroom components to the bottom of the template by adding a template override
- ...but couldn't work out how to remove the labels, not too sure on the content type of the page components, tried editing the fields and display but nothing too obvious on how to achieve this

### CODE REVIEW

**Comment by kurtisRogers-Nomensa** [comment]
@Polynya @siliconmeadow need one of you chaps to checkout ECCI-464, I got so far but couldn't work out the removal of the labels! I would like to know how to do this tho! Cheers :100:

File: themes/custom/essex/templates/content/node--localgov-newsroom--full.html.twig [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+        </div>
+
+        {{ content.localgov_page_components }}
+
+      </div>
+  </div>
+
+</article>
+
+{{ dump(content) }}
```
Comment (Polynya) [inline themes/custom/essex/templates/content/node--localgov-newsroom--full.html.twig:131]: Remove

**Comment by AnthonySGeorge** [comment]
@kurtisRogers-Nomensa  @Polynya  Should we close this MR until the label issue is sorted?

**Comment by siliconmeadow** [comment]
Tracking down that hiding of label thing was a pain, but got there. I didn't notice this PR, so I'd worked on a separate one in #282.

**Pull request closed without merging**